src/services/functionAppService.test.ts

```diff
@@ -282,13 +283,18 @@ describe("Function App Service", () => {
     });
   });
 
-  it("uploads functions to function app and blob storage", async () => {
-    const scmDomain = app.enabledHostNames.find((hostname) => hostname.endsWith("scm.azurewebsites.net"));
-    const expectedUploadUrl = `https://${scmDomain}/api/zipdeploy/`;
-
+  it("publishes functions and uploads to blob storage", async () => {
     const service = createService();
     await service.uploadFunctions(app);
-
+    const expectedArtifactName = service.getDeploymentName().replace("rg-deployment", "artifact");
+    expect((AzureBlobStorageService.prototype as any).uploadFile).toBeCalledWith(
+      slsService["artifact"],
+      configConstants.deploymentConfig.container,
+      `${expectedArtifactName}.zip`,
+    )
+    const uploadCall = ((AzureBlobStorageService.prototype as any).uploadFile).mock.calls[0];
+    expect(uploadCall[2]).toMatch(/.*-t([0-9]+)/);
+    const expectedUploadUrl = `https://${app.enabledHostNames[1]}/api/zipdeploy`;
```

Comment:
previous iteration hsa to find the scmDomain, why is it no longer needed here?

Reply:
nit: rename to `expectedPublishUrl` since it could be confuse with blob storage upload

Reply:
It's actually created inside the `mockFactory`, and is still the `scmDomain`